.travis.yml

```diff
@@ -140,13 +140,14 @@ script:
   # ctest -I <start>,<end>,<stride>,<list of test numbers>
   # 1 = common_tests
   # 2 = core tests
-  # 3 = application api tests
-  # 4 = shared_library tests
-  # 5 = helics-apps tests
-  # 6 = travis-tests
+  # 3 = key_tests
+  # 4 = application api tests
+  # 5 = extended_api tests
+  # 6 = shared_library tests
+  # 7 = helics-apps tests
   - ctest --verbose --timeout 120 -I 0,0,0,1
   # - ctest --verbose --timeout 120 -I 0,0,0,2
-  - ctest --verbose --timeout 120 -I 0,0,0,6
+  - ctest --verbose --timeout 120 -I 0,0,0,3
```

Comment:
There probably isn't a significant advantage to this on Travis/Appveyor, but if you're okay with the same timeout being applied to each test, the command could be changed to `ctest --verbose --timeout 120 -I 0,0,0,1,3` to run tests 1 and 3.  On Bamboo, one set of tests timed out and then it continued on with the next set of tests -- it seems to make getting dashboard results easier, since otherwise the different ctest runs overwrite the report file that gets output.

Reply:
I will probably tweak it a little more later once we have a better idea of typical run times.  I suspect the common tests should complete pretty quickly so that one might be reduced.  The application_api test probably require a little longer than 120 s, but I imagine it would be platform dependent